Project.toml

```diff
@@ -0,0 +1,13 @@
+name = "ImageMorphology"
+uuid = "8c0af99c-5f10-11e9-23dc-3302b17e12f9"
+version = "0.1.2"
+
+[deps]
+ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
+
+[extras]
```

Comment:
Should I add `[compat]` items?

If the answer is yes, is it?

```text
[compat]
ImageCore = "0.7"
```

Reply:
`"0.7.4"` if your `REQUIRE` edit was on purpose